components/eam/src/physics/rrtmgp/radiation.F90

```diff
@@ -809,7 +809,7 @@ subroutine radiation_init(state)
          end if
       end do
 
-      call addfld('EMIS', (/ 'lev' /), 'A', '1', 'Cloud longwave emissivity')
+      call addfld('EMIS', (/ 'lev' /), 'A', '1', 'Cloud 10.5 micron emissivity')
```

Comment:
maybe "10.5 micron (longwave) emissivity" for folks that don't think about wavelengths all the time?

Reply:
Or "longwave (10.5 micron) emissivity"? I can make that change, and a corresponding change for the SW properties as well. Just makes the names longer :-)

Reply:
I like "longwave (10.5 micron) emissivity". Might be longer, but it is just the "long name" field and it is oh so much more descriptive. And good idea to do the same thing for SW.